assets/models/system/Llama-2-7b-chat/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/Llama-2-7b-chat/1715241678/mlflow_model_folder
+  container_path: huggingface/Llama-2-7b-chat/1718291837/mlflow_model_folder
```

Comment:
reason why we are changing container? did the model weights change?

Reply:
uploaded finetune config to older folder and deleted new one